.rubocop.yml

```diff
@@ -5,11 +5,11 @@ require:
 
```

Comment:
The formatting was done by an extension.
If you feel that it's not useful, I'll revert it.